src/node_modules/dbpedia/ontology.mjs

```diff
@@ -0,0 +1,43 @@
+import { promises as fs } from 'fs';
+
+import * as _ from 'lamb';
+
+const FILE_ONTOLOGY_JSON = 'data/dbpedia/data/ontology.json';
+
+const dfs = (node, depth, { squash=false, fullURI=false } = {}) => {
+	const className = fullURI
+		? `http://dbpedia.org/ontology/${node.class}`
```

Comment:
I'd reuse `dbr` here and elsewhere

Reply:
Whoops, misread the URI base sorry, I see now that it's `dbr` but `dbo`. 

I'd have that value in a const for ease its detection later and facilitate reuse (possibly in `src/node_modules/sparql/util.mjs`)